rero_ils/modules/loans/jsonschemas/loans/loan-ils-v0.0.1.json

```diff
@@ -94,6 +94,10 @@
       "type": "string",
       "title": "Request pickup location PID"
     },
+    "selfcheck_terminal_id": {
+      "type": "string",
+      "title": "Selfcheck terminal id"
+    },
```

Comment:
Is this information could be more generlist ? Maybe in the future, we would like to know which operator initiate a loan (Checkout). Maybe I will rename this field "operator_id"... maybe create a fake `$ref` for terminal (terminal is not a ressource, it's just a DB reference). What's opinion of other devs ?

Reply:
`operator_id` already exist as `transaction_user_pid` in loan schema